packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/modeledDataStreamExplorer/useModeledDataStreams/useModeledDataStreams.ts

```diff
@@ -4,19 +4,20 @@ import invariant from 'tiny-invariant';
 
 import { ListModeledDataStreamsRequest } from './listModeledDataStreamsRequest';
 import { ModeledDataStreamCacheKeyFactory } from './modeledDataStreamCacheKeyFactory';
+import { SelectedAssetsPropsArray } from '../../types';
 
 export interface UseModeledDataStreamsProps {
-  assetIds: string[];
+  assetProps: SelectedAssetsPropsArray;
   client: IoTSiteWiseClient;
 }
 
-export function useModeledDataStreams({ assetIds, client }: UseModeledDataStreamsProps) {
+export function useModeledDataStreams({ assetProps, client }: UseModeledDataStreamsProps) {
   const cacheKeyFactory = new ModeledDataStreamCacheKeyFactory();
   const queries =
     useQueries({
-      queries: assetIds.map((assetId) => ({
+      queries: assetProps.map((assetProperties) => ({
```

Comment:
You called the item `assetProperties`, yet it does not appear we are iterating over asset properties.